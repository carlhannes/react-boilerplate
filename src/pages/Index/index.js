import React from 'react';

import Layout from '../../layout';

import Sidebar from '../../components/Sidebar';

function Index() {
  return (
    <Layout sidebar={(<Sidebar />)}>
      <h1>Index</h1>
    </Layout>
  );
}

export default Index;
