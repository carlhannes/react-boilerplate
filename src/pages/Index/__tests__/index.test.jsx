import React from 'react';
import { mount } from 'enzyme';

import Index from '../index';

jest.mock('../../../components/Sidebar/sidebar');

describe('Index page', () => {
  it('should render correctly', () => {
    const wrapper = mount(<Index />);
    wrapper.update();

    expect(wrapper).toMatchSnapshot();
  });
});
