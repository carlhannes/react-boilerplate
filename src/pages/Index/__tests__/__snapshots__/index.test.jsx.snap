// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page should render correctly 1`] = `
<Index>
  <Layout
    sidebar={<Sidebar />}
  >
    <div
      className="sidebar"
    >
      <Sidebar>
        <ul>
          <li>
            sidebar mock
          </li>
        </ul>
      </Sidebar>
    </div>
    <main>
      <h1>
        Index
      </h1>
    </main>
  </Layout>
</Index>
`;
